OrbitAccessibility/include/OrbitAccessibility/AccessibleInterfaceRegistry.h

```diff
@@ -0,0 +1,45 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_ACCESSIBILITY_ACCESSIBLE_INTERFACE_REGISTRY_H_
+#define ORBIT_ACCESSIBILITY_ACCESSIBLE_INTERFACE_REGISTRY_H_
+
+#include <functional>
+
+#include "AccessibleInterface.h"
```

Comment:
I would have expected this to have to be `#include "OrbitAccessibility/AccessibleInterface.h", otherwise it would cause problems when this file is included from another module. But I guess I'm wrong, I'm a bit lost here.

Reply:
Hm true, I guess it works because AccessibleInterface.h is in the same folder, I've changed it for consistency.